tests/integration/test_nitrate.py

```diff
@@ -278,3 +278,26 @@ def test_basic(self):
             generated = yaml.load(file)
             referenced = yaml.load(self.main_fmf_content)
             self.assertEqual(generated, referenced)
+
+    def test_old_relevancy(self):
+        os.chdir(self.tmpdir / "import_old_relevancy")
+        files = os.listdir()
+        self.assertEquals(files, ['Makefile'])
+        runner = CliRunner()
+        self.runner_output = runner.invoke(
+            tmt.cli.main, [
+                "test", "import", "--nitrate"], catch_exceptions=False)
+        self.assertEqual(self.runner_output.exit_code, 0)
+
+        tree_f36_intel = tmt.Tree(
+            '.',
+            context={
+                'distro': ['fedora-36'],
```

Comment:
What about trying also with `fedora-35` to exercise the second relevancy rule as well?

Reply:
Sure, hopefully requre stays happy.